docs/release-notes/2.35.0.md

```diff
@@ -0,0 +1,134 @@
+
+### DBaaS: Backup schedule while creating DB clusters
+
+Starting with PMM 2.35.0, you can add a backup schedule while creating DB clusters in DBaaS. This feature is a fusion of backup management and DBaaS in PMM. Currently, DBaaS only supports scheduled backups, which can only be enabled when a database cluster is created.
+
+!!! note alert alert-primary "Note"
+    DBaaS does not support on-demand backups.
+
+### DBaaS: Restore cluster from backup stored on S3
+
+You can now restore a cluster from backup stored on S3 while creating DB clusters in DBaaS. Currently, you cannot restore a cluster from a backup. You can just create a database cluster from backup.
+
+!!! note alert alert-primary "Note"
+    DBaaS does not support on-demand-restore.
+
+### More pre-run checks for MongoDB backups
+With this release, PMM runs additional verifications to ensure infrastructure compliance and prevent backup execution errors:
+
+#### Tool compatibility check
+
+Before initiating backups for MongoDB, PMM now examines the current MongoDB node to ensure that the connected PMM Agents have the tools to execute and finalize the backup.
+
+If the requirements are not met, PMM reports what tools or tool versions are missing, similarly to the existing prerequisites check for MySQL backups.
+
+#### Overlapping  jobs check
+
+To avoid conflicting jobs, PMM proactively disables the creation of  Point-in-time Recovery (PITR), or snapshot backups, when other backups are scheduled for running.
+
+Similarly, creating PITR backups for different services of the same cluster is now automatically restricted.
+
+#### Cluster membership check
+
+To avoid execution errors, PMM no longer allows scheduling backups on MongoDB Services that are not managed as a cluster.
+
+You can check which of your current Services are not members of a cluster on the **PMM Inventory > Services** page. Services that do not specify a cluster name in the **Other Details** column should be removed and re-added using the following command:
+
+<pre>pmm-admin add mongodb
+username=pmm_mongodb --password=password 
+query-source=profiler <mark>--cluster=mycluster</mark></code></pre>
+
+
+### Backup restore logs accessible from PMM
+
+With this release, we are adding the option to check the logs for restores from a MongoDB database backup.
+This complements the existing option to check the logs for the backup process and makes it easier to gather data for compliance, auditing, and troubleshooting purposes.
+
+Instead of having to look through PBM log files, you can now access the **Log** option in PMM, which is now automatically displayed next to each MongoDB backup restore entry on the **Backup > Restores** page.
+
+### Components Upgrade
+
+MongoDB exporter has been upgraded to 0.37.0.
+
+## New Features
+
+- [PMM-10325](https://jira.percona.com/browse/PMM-10325) - Starting with PMM 2.35.0, for enhanced security, you can change the admin password even after `pmm-agent` is registered on the PMM Server.
```

Comment:
@rishat-ishbulatov @rasika-chivate this explanation doesnt give the story :) Could we add the problem and how we solved it in brief?

Reply:
Is the user really interested in how we solved the problem?

Reply:
Resolving this.